public/data/kartik2220.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Kartik Gajjar",
+  "type": "personal",
+  "bio": "I am Kartik, currently pursuing my Engg. majoring in computer science . I am a tech & open source enthusiast . I am always open to collaborating on projects .",
+  "avatar": "https://github.com/Kartik2220.png",
+  "links": [
+    {
+      "name": "Follow my open source on GitHub",
+      "url": "https://github.com/Kartik2220",
+      "icon": "github"
+    },
+     {
+      "name": "take a look at my website",
```

Looks good to me! 🚀 

What do you think of capitalizing this `t` here
```suggestion
      "name": "Take a look at my website",
```